src/utils/common_spaces.jl

```diff
@@ -100,8 +132,39 @@ function make_hybrid_spaces(
         z_mesh,
     )
     z_grid = Grids.FiniteDifferenceGrid(z_topology)
-    if isnothing(surface_warp)
+    if isnothing(surface_warp) && parsed_args["topography"] != "Earth"
         hypsography = Hypsography.Flat()
+    elseif isnothing(surface_warp) && parsed_args["topography"] == "Earth"
+        @info "SpaceVaryingInputs: Remapping orography onto spectral space"
+        z_surface = SpaceVaryingInputs.SpaceVaryingInput(
+            AA.earth_orography_file_path(;
+                context = ClimaComms.context(h_space),
+            ),
+            "z",
+            h_space,
+        )
+        parent(z_surface) .=
+            ifelse.(parent(z_surface) .< FT(0), FT(0), parent(z_surface))
```

Comment:
Maybe we should apply this to the input data directly? You can pass a similiar preprocess func to the `SpaceVaryingInput`

Reply:
Same as above with regards to avoiding using `parent`